app/views/invitation_mailer/invite_with_enroll.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="container">
+  <p>
+    <%= "Hi #{@email}," %>
+  </p>
+  <p>We are happy to inform you that you have been accepted to the Turing School of Software & Design. Congratulations and welcome!</p>
+  <p><strong>Please follow <%= link_to("this link", @url) %></strong> to create an account with Census, the Turing School of Software & Design's identity management application.
+  </p>
+  <p><strong>Next, please visit <a href='https://enroll.turing.io/'>enroll.turing.io</a></strong> to find more information about our programs, enrollment availability, and to pick a starting date.</p>
+  <p>We invite you to explore the following financing options available to our students:</p>
+  <ul>
+    <li><%= link_to("Sallie Mae", "https://www.salliemae.com/ctsmartloan") %></li>
+    <li><%= link_to("SkillsFund", "http://turing.skills.fund/accepted") %></li>
+    <li><%= link_to("Climb",      "http://www.climbcredit.com/apply/turing") %></li>
+  </ul>
+  <p>These options make it possible for you to start at Turing for as little as $1,000 up front and finance the rest. If you are interested in financing living expenses, you can do so for up to $14,000 with Sallie Mae or SkillsFund and $3500 with Climb. For students unable to secure outside financing, Turing offers alternative payment plans.</p>
```

Comment:
Thoughts on making this <p> span two lines so there aren't so many characters in one line?

Reply:
Works for me! That shouldn't muck up the HTML, right?

Reply:
I don't think so. I was second guessing myself though so I'll go double check.

Reply:
Yep, it won't screw things up. The `<p>` ignores any line breaks in your editor.